lib/virt_autotest/utils.pm

```diff
@@ -447,10 +457,10 @@ sub wait_guests_shutdown {
     # Note: Domain-0 is for xen only, but it does not hurt to exclude this also in kvm runs.
     # Firstly wait for guest shutdown for a while, turn it off forcibly using "virsh destroy" if timed-out.
     # Then wait for guest shutdown again with default "die => 1".
-    if (script_retry("! virsh list | grep -v Domain-0 | grep running", delay => 1, retry => $retries, die => 0) ne '0') {
+    if (script_retry("! virsh list | grep -v Domain-0 | grep running", timeout => 60, delay => 1, retry => $retries, die => 0) != 0) {
```

Comment:
Would you please explain why timeout is needed when retry is customized and delay fixed?

Reply:
Command had ever failed due to timed out.

Reply:
$timeout is for each execution of 'virsh list ...', rather than the total time of the script_retry().

Reply:
I double checked, this is necessary. This 'timeout' will be used in script_retry as below:
```
script_retry("timeout $timeout $cmd", $timeout + 3, xxxx)
```